janus_core/cli.py

```diff
@@ -127,6 +130,8 @@ def singlepoint(
         Keyword arguments to pass to the selected calculator. Default is {}.
     write_kwargs : Optional[dict[str, Any]]
         Keyword arguments to pass to ase.io.write when saving results. Default is {}.
+    log_file : Optional[str]
```

Comment:
Does that want to be a path rather than a string?

Reply:
At the very least we may want to set up a type-alias to indicate where strings should be pathy

Reply:
Ah thought this was in single_point.py.

Typer doesn't support `Union` types yet, annoyingly, so I've changed it to `Path` here, and `PathLike` in single_point.py